cmds/ocm/commands/ocmcmds/common/resources.go

```diff
@@ -443,12 +446,12 @@ func (o *ResourceAdderCommand) ProcessResourceDescriptions(listkey string, h Res
 	defer obj.Close()
 
 	for _, r := range resources {
-		ictx := ictx.Section("adding %s...", r.Spec().Info())
+		isctx := ictx.Section("adding %s...", r.Spec().Info())
```

Comment:
The idea here was, to always have an input context (`ictx`) available, using the same name, regardless of the nesting level.

Therefore at the beginning of a nesting level a new local variable with the same name is created.
This makes the code look the same regardless of the level.

Is this renaming really required by the new linter?

Reply:
Yes, and it can be misleading as well, not just for linters, but in general. Shadowing a variable can be tricky and annoying, because as a reader (or someone who wants to change something), they would expect it to be exactly the same, but it's not, it's a new input context with a new section (based on the previous input context).

Reply:
I could also argument exactly the other way round. If I use the input context is several blocks, I always want to use the input context applicable for this block. If I always have to use another variable, depending on the block, the code is in, then this would be very error prone.

But I agree, that in most cases you are right. But as always, it depends on the intended use case.